english/drool/droolstrategem.md

```diff
@@ -1,7 +1,13 @@
 You casually hold your face over the outstretched hand and protrude your tongue over the offending appendage.
 
-As the saliva flows freely over your taste buds you contemplate what decisions in life have lead you to this point.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.